docs/content/developer-workflow.md

```diff
@@ -0,0 +1,302 @@
++++
+title= "Source Code Organization and Your Development Workflow"
+date= 2017-12-01T18:01:57-08:00
+description = ""
+draft= true
++++
+
+# Source Code Organization and Your Development Workflow
+
+You've made a Hello World function in your favourite language, and
+you've run it on your Fission deployment.  What's next?
+
+How should you organize source code when you have lots of functions?
+How should you automate deployment into the cluster?  What about
+version control?  How do you test before deploy?
+
+The answers to these questions start from a common first step: how do
+you _specify an application_?
+
+## Declarative Specifications
+
+You can specify your functions in a set of YAML files instead of
+invoking the Fission CLI commands.
+
+For applications that involve many functions, this is better than
+scripting the `fission` CLI, which is meant as a user interface, not a
+programming interface.
+
+You'll usually want to track these YAML files in version control along
+with your source code.  Fission provides CLI tools for generating
+these specification files, validating them, and "applying" them to a
+Fission installation.
+
+What does it mean to _apply_ a specification?  It means putting
+specification to effect: figuring out the things that need to be
+changed on the cluster, and changing them to make them the same as the
+specification.
```

Comment:
Change to: "changed on the cluster, and updating them to match the specification."